Formula/qt.rb

```diff
@@ -66,6 +67,9 @@ class Qt < Formula
   def install
     resources.each { |addition| addition.stage buildpath/addition.name }
 
+    # force cmake to use GSS.framework
+    inreplace "qtbase/cmake/FindGSSAPI.cmake", "gssapi_krb5", ""
```

Comment:
What's the difference between doing this and using `CMAKE_FIND_FRAMEWORK=FIRST`?

Reply:
Homebrew's `std_cmake_args` uses `-DCMAKE_FIND_FRAMEWORK=LAST`.
If uses `-DCMAKE_FIND_FRAMEWORK=FIRST`, framework `QtGui` will link against an unexpected `libpng`, and for network API, it will use `gssapi_krb5`, this library doesn't contain `qt` required symbols.
See https://github.com/qt/qtbase/blob/dev/cmake/FindGSSAPI.cmake and https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_FRAMEWORK.html

Reply:
Can we change it to `CMAKE_FIND_FRAMEWORK=NEVER`?

Reply:
`qt` need `IOKit`.

Reply:
Sorry, a mistake in previous reply, when use `-DCMAKE_FIND_FRAMEWORK=LAST` `qt` will use `gssapi_krb5` which doesn't contain qt required symbols.

Reply:
I'm confused then -- why does it link with `libpng` when using `CMAKE_FIND_FRAMEWORK=FIRST`? `libpng` is not a framework?

Reply:
When using `CMAKE_FIND_FRAMEWORK=FIRST`, `cmake` will find `libpng` in `/Library/Frameworks/Mono.framework`, which version is 1.14.2.

Reply:
So why don't we want it to link with `libpng` there?

Reply:
The header doesn't match the library, some application in `qt` will report `Application built with libpng-1.4.12 but running with 1.6.37`

Reply:
It seems related to the CI environment. I will close it. 😅